### DISCUSSION TITLE
Some scores accuracy displayed varies by 0.01% compared to stable

### DISCUSSION THREAD

**Discussion by hiderikzki** [ppy/osu#30824] [Q&A]
### Type

Game behaviour

### Bug description

Some scores seen on leaderboard have an accuracy lower compared to stable by 0.01% (possibly due to wrong rounding??)

Check examples from images 

Map used for example: [Fleshgod Apocalypse - In Aeternum [Benedict Anima Mea]](https://osu.ppy.sh/beatmapsets/1261793#osu/2622787)

### Screenshots or videos

Stable:
![image](https://github.com/user-attachments/assets/98539ca8-c810-4930-8c6c-a4e283aaf268)

Lazer:
![image](https://github.com/user-attachments/assets/59758993-2e05-429d-9e68-c83d11440e7e)

### Version

2024.1115.3

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17852515/compressed-logs.zip)

**Comment by smoogipoo** [comment]
Yes, this is caused by rounding differences, because lazer rounds down to avoid showing (in other cases) 100% when scores are >99.995%.

https://github.com/ppy/osu/blob/6a0ac4c29eca22bdfe134cdd5e51395bbd6e1d49/osu.Game/Utils/FormatUtils.cs#L13-L27

Not sure if this needs to be addressed - if anything it's wrong on osu!stable and osu!web instead of lazer.
